dask/dataframe/core.py

```diff
@@ -3122,13 +3122,24 @@ def nunique(self, split_every=None):
 
     @derived_from(pd.Series)
     def value_counts(
-        self, sort=None, ascending=False, dropna=None, split_every=None, split_out=1
+        self,
+        sort=None,
+        ascending=False,
+        dropna=None,
+        normalize=False,
+        split_every=None,
+        split_out=1,
     ):
         """
         Note: dropna is only supported in pandas >= 1.1.0, in which case it defaults to
         True.
         """
-        kwargs = {"sort": sort, "ascending": ascending}
```

Comment:
It didn't seem useful to pass these to each chunk, but I can take this change out of this PR if it is too unrelated.

Reply:
Hmm - Good point.  It may not be necessary to pass these until aggregation (as you are doing here).

Reply:
Something got messed up with the `ddf.describe` with this case, so I went back on it.